SDK/AppCenter/Microsoft.AppCenter.WindowsDesktop/Utils/Constants.cs

```diff
@@ -32,5 +35,17 @@ public static string AppCenterFilesDirectoryPath
         public static string AppCenterDatabasePath = Path.Combine(AppCenterFilesDirectoryPath, "Logs.db");
 
         public static string LocalAppData => Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData);
+
+        public static string UserName
```

Comment:
I guess something like
```suggestion
        public static string UserName { get; internal set; } = Environment.UserName;
```
should work well here